test/e2e/apimachinery/get_apigroup_list.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
nit: s/2019/2020/

Reply:
Fixed. Renamed file to `apigroup_list.go`